### DISCUSSION TITLE
Resume Review ×ATS Scoring

### DISCUSSION THREAD

**Discussion by Garima-149** [recodehive/resume-pitch#87] [Q&A]
Name - Garima
Linked In profile- https://www.linkedin.com/in/garima-chandra-476377288?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app

Github profile- https://github.com/Garima-149/

Resume link- https://drive.google.com/file/d/1mMML9J_oMZGwQIaCdjOi4rZthFHy54r4/view?usp=drivesdk

**Comment by sanjay-kv** [comment]
[garima.pdf](https://github.com/user-attachments/files/17089323/garima.pdf)
[ATS Version of CV (24).docx](https://github.com/user-attachments/files/17089325/ATS.Version.of.CV.24.docx)

@Garima-149
